src/django/api/models/facility/facility_claim.py

```diff
@@ -6,6 +6,7 @@
 from django.db import models
 
 from countries.lib.countries import COUNTRY_CHOICES
+from api.constants import FacilityClaimStatuses
 from ...constants import Affiliations, Certifications
```

Comment:
Please fix relative imports

Reply:
This will be fixed in scope of this ticket: https://opensupplyhub.atlassian.net/browse/OSDEV-1143